kivy/uix/button.py

```diff
@@ -12,6 +12,9 @@
 
     button = Button(text='Hello world', font_size=14)
 
+.. note:: To fit the Button to its :attr:`~Button.text`, bind size to :attr:`~kivy.uix.label.Label.texture_size`
```

Comment:
Ok, what do you think about expanding the last sentence instead?

Currently:
> To configure the button, the same properties are used as for the Label class:

Change to:

> To configure the button, the same properties (padding, font_size, etc) and sizing system are used as for the Label class: